src/search.rs

```diff
@@ -109,7 +149,7 @@ async fn search_subreddits(q: &str, typed: &str) -> Vec<Subreddit> {
 			let icon = subreddit["data"]["community_icon"].as_str().map_or_else(|| val(subreddit, "icon_img"), ToString::to_string);
 
 			Subreddit {
-				name: val(subreddit, "display_name_prefixed"),
+				name: val(subreddit, "display_name"),
```

Comment:
Using `display_name` instead (with a corresponding change in search.html) because the alt-text looked like `r/r/subname`